engine/output.ftl

```diff
@@ -458,22 +499,43 @@
                                     [ SHARED_PROVIDER],
                                     UNIQUE_COMBINE_BEHAVIOUR
                                 ),
-                                subset
+                                subset,
+                                converter
                             )]
 
     [#local outputSuffix = (outputMappings["OutputSuffix"])!"" ]
 
     [#return formatName(outputPrefix, outputSuffix)]
 [/#function]
 
-[#function getGenerationContractStepParameters subset alternative ]
+[#function getGenerationContractProperties ]
+    [#return
+        {
+            "entrance"               : getCLOEntranceType(),
+            "flows"                  : getCLOFlows()?join(","),
+            "providers"              : (getLoaderProviders()?join(","))!SHARED_PROVIDER,
+            "deploymentFramework"    : getCLODeploymentFramework(),
+            "deploymentUnit"         : getCLODeploymentUnit(),
+            "deploymentGroup"        : getCLODeploymentGroup(),
+            "account"                : getCommandLineOptions().Layers[ACCOUNT_LAYER_TYPE],
+            "accountRegion"          : contentIfContent(getCLOAccountRegion(), getAccountLayerRegion()),
+            "region"                 : contentIfContent(getCLOSegmentRegion(),getProductLayerRegion()),
```

Comment:
Is this just interim? With multi-placement, the account/region can be different per resourceGroup so better to leave on the pass. Same for region.

Reply:
I went with reality, we can just move it when that is the case

Reply:
Ok - I expect that will be the case for the placement stuff but happy for this to go through for now.